.travis.yml

```diff
@@ -59,7 +59,7 @@ script:
   # Test ruby gem
   - gem install gem-path --no-ri --no-rdoc
   - cd $HOME/gems/symengine-0.1.0/
-  - bundle exec rspec
+  - bundle exec rspec --require ./spec/spec_helper.rb
```

Comment:
I see. Since we are already including the `spec/` folder, I guess there is no harm in including `.rspec` too. By the way, should `spec/` folder be packaged with the gem?